--- /home/mark/converters.py.orig	2009-12-20 00:47:58.000000000 -0500
+++ ./converters.py	2009-12-20 00:48:16.000000000 -0500
@@ -42,8 +42,8 @@
             type_codes.WRD: int,
             type_codes.BIGINT: int,
             type_codes.SERIAL: int,
-            type_codes.REAL: int,
-            type_codes.DOUBLE: int,
+            type_codes.REAL: float,
+            type_codes.DOUBLE: float,
             type_codes.BOOLEAN: self.__bool,
             type_codes.DATE: self.__date,
             type_codes.TIME: self.__time,
